code/style.css

```
@@ -1,7 +1,497 @@
+
+p, span, h1, h2, h3 {
+  -webkit-font-smoothing: antialiased;
+}
+
+h1, h2, h3 {
+  font-family: 'Roboto Mono', monospace;
+  font-style: normal;
+  font-weight: bold;
+  margin-top: 0;
+}
+
+h1 {
+  font-size: 42px;
+  line-height: 55px;
+}
+
+h2 {
+  font-size: 30px;
+  line-height: 40px;
+  margin-bottom: 16px;
+}
+
+.nav-links > a {
+  margin-left: 24px;
+}
+
+a {
+  font-family: 'Lexend Peta', sans-serif;
+  text-decoration: none;
+  color: rgba(255,255,255,0.7);
+  font-size: 14px;
+  text-transform: uppercase;
+}
+a:hover {
+  color: rgba(255,255,255,0.85);
+}
+
+.emphasise {
+  font-style: italic;
+}
+
+
+/* select h2 in a class card-full */
+.card-full > h2  {
+  margin: 0 auto 16px 16px;
+}
+
+.card-full {
+}
+
+h3 {
+  font-size: 24px;
+  line-height: 32px;
+}
+
+p {
+  font-family: 'Roboto Slab', serif; 
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  
+}
+
+.p-small {
+  font-family: 'Roboto Slab', serif; 
+  font-size: 16px;
+}
+
+.smaller {
+  font-size: 14px;
+  color: rgba(255,255,255,0.7);
+}
+
+.smaller a {
+  font-family: 'Roboto Slab', serif; 
+  color: #5DC3CA;
+  text-transform: none;
+  font-size: 14px;
+}
+
+/* ===== form input styling ===== */
+
+label {
+  margin: 0 0 8px 16px;
+}
+
+.bold {
+  font-weight: 700;
+}
+
+input[type=text], input[type=email] {
+  border-radius: 24px;
+  border: 1px solid rgba(0,0,0,0.20);
+  padding: 10px 24px;
+
+  font-family: 'Roboto Slab', serif; 
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 24px;
 }
+
+input[type=text]:focus, input[type=email]:focus {
+  border: 1px solid #BF85BF;
+  outline-width: 0;
+  box-sizing: border-box;
+  box-shadow: inset 0 0 0px 2px #BF85BF;
+}
+
+.slider {
+  -webkit-appearance: none;
+  appearance: none;
+  height: 8px;
+  background-color: rgba(0,0,0,0.11);
+  border-radius: 100px;
+  width: 80%;
+}
+
+.slider:focus {
+  outline-width: 0;
```

This is accessibility style we shouldn't get rid of, that's an antipattern. You can always change default style, but not hide it completely